src/uwtools/drivers/forecast.py

```diff
@@ -40,33 +50,87 @@ def validate(self):
 
     def create_model_config(self):
 
-        ''' Collects all the user inputs required to create a model config 
+        ''' Collects all the user inputs required to create a model config
         file, calling the existing model config tools. This will be unique
         to the app being run and will appropriately parse subsequent stages
-        of the workflow. 
-        Defaults will be filled in if not provided by the user. Equivalent 
+        of the workflow.
+        Defaults will be filled in if not provided by the user. Equivalent
         references to config_default.yaml or config.community.yaml from SRW
         will need to be made for the other apps. '''
 
     def stage_fix_files(self):
 
-        ''' Holds the knowledge for how to modify a list of fix files and 
-        stages them in the working directory. Likely gets all its info from 
-        config_obj. Calls data mover tool (could be python copy). Fix files 
+        ''' Holds the knowledge for how to modify a list of fix files and
+        stages them in the working directory. Likely gets all its info from
+        config_obj. Calls data mover tool (could be python copy). Fix files
         usually are specific to a given named grid and resolution. '''
 
     def create_namelist(self):
-        ''' Collects all the user inputs required to create a namelist 
+        ''' Collects all the user inputs required to create a namelist
         file, calling the existing namelist config tools. For SRW,
         this will take the fix files and model_config previously created to
         to generate config.yaml'''
 
+    def create_directory_structure(self, run_directory, exist_act="delete"):
```

Comment:
At some point, there was a 'stage input files' but that got lost along the way it seems - thanks for catching that and adding this!